py-scripts/lf_ftp.py

```diff
@@ -2311,6 +2311,67 @@ def generate_dataframe(self, groupdevlist: List[str], clients_list: List[str], m
         else:
             return None
 
+    def monitor_cx(self):
```

Comment:
Same comment as other PR wrt `monitor()` function. If it doesn't exist for L4 endpoint type, maybe it would be good to add it for L4 CXProfile instead?